R/pkg/R/DataFrame.R

```diff
@@ -1174,7 +1174,7 @@ setMethod("dim",
 #' firstName <- names(collected)[1]
 #' }
 #' @note collect since 1.4.0
-setMethod("collect",
+setMethod("collect", # nolint
```

Comment:
there are so many this kind of things. Can we adjust complexity_limit or disable cyclocomp_linter? I assume this nolint is for cyclocomp_linter.

Reply:
Yes. This is about the length of functions which we don't respect. Let me try, @viirya .

Reply:
BTW, we also are going to lose the controllability for new codes (PRs), too. Is it okay?

Reply:
as there are so many violations, seems to me it is not really an issue to us.